nvflare/fuel/hci/client/cli.py

```diff
@@ -348,8 +374,7 @@ def cmdloop(self, intro=None):
                         except EOFError:
                             line = "bye"
                         except KeyboardInterrupt:
-                            self.stdout.write("\n")
-                            line = "\n"
+                            return
```

Comment:
why are we removing these statements?

Reply:
What is the behavior we want on KeyboardInterrupt? I changed it to exit when catching a KeyboardInterrupt while waiting at the input() call since I had the thread raise signal.SIGINT when the session was expired.